### ISSUE TITLE
Do you have a plan to add examples in PHP?

### ISSUE THREAD

**Issue by NabiKAZ** [coinexcom/coinex_exchange_api#80]
Do you have a plan to add examples in PHP?
Do you want me to work on it?

**Comment by kendol07** [comment]
in my repositories I have some examples in PHP I recommend that you go to see them https://github.com/kendol07/Coinex-API-PHP

**Comment by NabiKAZ** [comment]
@kendol07 Yes I was see, I means a official codes in this repository inside another languages.

**Comment by NabiKAZ** [comment]
@coinexcom @buyongji

I started a standard PSR-4 project under composer in here:
https://github.com/NabiKAZ/Coinex-API-PHP

And so summited a PR for merge with current coinex project.

**Comment by zhouaini528** [comment]
@NabiKAZ Hope for your individual [https://github.com/zhouaini528/coinex-php](https://github.com/zhouaini528/coinex-php)
